ksql-common/src/test/java/io/confluent/ksql/util/KsqlConfigTest.java

```diff
@@ -68,20 +76,81 @@ public void shouldNotSetDeserializationExceptionHandlerWhenFailOnDeserialization
   }
 
   @Test
-  public void shouldSetStreamsConfigProperties() {
+  public void shouldSetStreamsConfigConsumerUnprefixedProperties() {
     final KsqlConfig ksqlConfig = new KsqlConfig(Collections.singletonMap(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "100"));
     Object result = ksqlConfig.getKsqlStreamConfigProps().get(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG);
     assertThat(result, equalTo("100"));
   }
 
+  @Test
+  public void shouldSetStreamsConfigConsumerPrefixedProperties() {
+    final KsqlConfig ksqlConfig = new KsqlConfig(Collections.singletonMap(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "100"));
+    Object result = ksqlConfig.getKsqlStreamConfigProps().get(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG);
+    assertThat(result, equalTo("100"));
+  }
+
+  @Test
+  public void shouldSetStreamsConfigProducerUnprefixedProperties() {
+    final KsqlConfig ksqlConfig = new KsqlConfig(
+        Collections.singletonMap(
+            StreamsConfig.CONSUMER_PREFIX + ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "100"));
+    Object result = ksqlConfig.getKsqlStreamConfigProps().get(
+        StreamsConfig.CONSUMER_PREFIX + ConsumerConfig.AUTO_OFFSET_RESET_CONFIG);
+    assertThat(result, equalTo("100"));
+  }
+
+  @Test
+  public void shouldSetStreamsConfigProducerPrefixedProperties() {
+    final KsqlConfig ksqlConfig = new KsqlConfig(
+        Collections.singletonMap(
+            StreamsConfig.PRODUCER_PREFIX + ProducerConfig.BUFFER_MEMORY_CONFIG, "1024"));
+    final Object result = ksqlConfig.getKsqlStreamConfigProps().get(
+        StreamsConfig.PRODUCER_PREFIX + ProducerConfig.BUFFER_MEMORY_CONFIG);
+    assertThat(result, equalTo(1024L));
+  }
+
+  @Test
+  public void shouldSetStreamsConfigAdminClientProperties() {
+    final KsqlConfig ksqlConfig = new KsqlConfig(
+        Collections.singletonMap(AdminClientConfig.RETRIES_CONFIG, 3));
+    final Object result = ksqlConfig.getKsqlStreamConfigProps().get(
+        AdminClientConfig.RETRIES_CONFIG);
+    assertThat(result, equalTo(3));
+  }
+
+  @Test
+  public void shouldSetStreamsConfigProperties() {
+    final KsqlConfig ksqlConfig = new KsqlConfig(
```

Comment:
This seems identical to the test on line 86.

Reply:
Good catch. I meant for this to be an actual streams config (e.g. cache.max.bytes.buffering)